packages/theming/src/theme/index.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Similar to `palette`, could we combine this into a single file to help with readability? It will be large, but would be useful to understand the relations between the different values.